### ISSUE TITLE
How to stop a Countdown when the App is backgrounded (Xamarin forms)

### ISSUE THREAD

**Issue by dr1sshamra** [Red-Folder/bgs-core#98]
Hello everyone, i develop a Quizz App , i used Device.StartTimer to run my countdown and its works fine,
also i added Sounds for the countdown, the problem is when The App is Backgrounded, the Countdown and the sounds still running. thats leading the app to be crashed. Any solution to stop the countdown/sounds on sleeps mode ?
Sorry for bad english.

**Comment by Red-Folder** [comment]
Is this a Xamarin forms question?

I wouldn't expect this library to work with Xamarin forms

**Comment by dr1sshamra** [comment]
Yes !! it is . and it works good !! i just need to stop the countdown when the app is backgrounded !
